news.yml

```diff
@@ -9,3 +9,8 @@
   content: |
     Náš první workshop programování pro začátečnice v Hradci Králové je tu! Konat se bude 25. března a
     registrovat se na něj <a href="https://docs.google.com/forms/d/e/1FAIpQLScO5in0gzo13ddLcc1q508beqc5RGp6Dpd-3XGGP9HrF_47rg/viewform" target="_blank">můžeš zde</a>
+
+- title: Přihlašování na jarní začátečnické kurzy v Pythonu
+  expires: 2018-03-01
+  content: |
+    V Praze  právě teď chystáme nové tříměsíční kurzy Pythonu pro úplné začátečnice! Dva kurzy nyní poběží souběžně, a to od začátku od března, více informací v přihlášce <a href="https://goo.gl/forms/tH2mK2PsBLqODYKn1" target="_blank">na tomto odkazu</a>.
```

Comment:
od začátku března?

Reply:
jo, do 1.3. je přihlašování a o týden později se začne, to pořád považuju za začátek března. :) Je to v té přihlášce napsané.

Reply:
tak detaily si snad lidi přečtou v přihlášce...

Reply:
Jakože tam máš asi navíc to druhé slovo od

Reply:
jo tak, to mi uteklo, dík!